static/app/components/accordion/accordionSummary.tsx

```diff
@@ -0,0 +1,38 @@
+import {useState} from 'react';
+import styled from '@emotion/styled';
+
+import {IconChevron} from 'sentry/icons';
+
+export interface AccordionSummaryProps extends React.HTMLAttributes<HTMLDivElement> {
+  onClick: () => void;
+  disabled?: boolean;
+}
+
+function AccordionSummary({onClick, disabled = false, children}: AccordionSummaryProps) {
```

Comment:
Commented above - doesn't feel like this component is necessary

Lets just pass `title` as a prop into the accordion (typed string or element) and leave this logic in the accordion component above